packages/toolpad-core/.npmrc

```diff
@@ -0,0 +1 @@
+enable-pre-post-scripts=true
```

Comment:
Do we need this? I see no pre or post scripts in the package.json. And we probably shouldn't rely on them anyway if the package manager doesn't support it out of the box. We'll do task orchestration with `nx` if we need to. X/core will be using it vor that purpose as well in the future.

Reply:
Changing buils script to ""build": "pnpm prebuild && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files" if that's better.